spot-electron/src/bt-beacon/daemon/win32/bluetoothd.js

```diff
@@ -30,6 +30,10 @@ class BluetoothD {
 
         this.advertiser.on('StatusChanged', ({ status }) => {
             this.status = status;
+            if (this.advertisingStatusUpdatedCallback === 'function') {
+                this.advertisingStatusUpdatedCallback(status === 2);
```

Comment:
this should probably be `status !== 2`?

Reply:
No, true equals running state, false equals stopped (or failed of any sort). See line 84, 85 below.

Reply:
I still think this code is too hard to understand, but I give up